components/user-store/org.wso2.carbon.identity.user.store.configuration/src/main/java/org/wso2/carbon/identity/user/store/configuration/utils/SecondaryUserStoreConfigurationUtil.java

```diff
@@ -320,6 +322,24 @@ public static String getUserStoreProperties(UserStoreDTO userStoreDTO) throws Id
         return userStoreProperties;
     }
 
+    public static Map<String, String> getSecondaryUserStorePropertiesFromTenantUserRealm(String userStoreDomain)
+            throws IdentityUserStoreMgtException {
+
+        Map<String, String> secondaryUserStoreProperties = null;
+        try {
+            UserStoreManager secondaryUserStoreManager = getSecondaryUserStoreManager(userStoreDomain);
+            if (secondaryUserStoreManager != null) {
+                secondaryUserStoreProperties = secondaryUserStoreManager.getRealmConfiguration()
+                        .getUserStoreProperties();
+            }
+        } catch (UserStoreException e) {
+            String errorMessage = "Error while retrieving user store configurations";
+            log.error(errorMessage, e);
```

Comment:
If the throwing message is not overriding from a higher level , we don't need to log and throw.

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/commit/97cd4b67b5b86fdf62f343d31c78b02f74d52063